BUILD.md

```diff
@@ -183,7 +183,7 @@ cmake -DLLVM_ENABLE_PROJECTS='clang' -DCMAKE_BUILD_TYPE=Debug '-DLLVM_TARGETS_TO
 ```
 
 ```
-make -j 32
```

Comment:
The original instructions just have `make`. I have been doing `make -j<number>` because I thought that made it multithreaded, but upon checking the documentation it actually just _limits_ the number of simultaneous jobs to that number. Make will automatically figure out the right number of jobs for you if you leave it off.

Reply:
It's for the same reason that I'm proposing to remove `-32` from the suggested `ninja` command. You're not telling it to use 32 threads, you're telling it _not_ to use _more_ than 32 threads, which seems pointless to me.

Reply:
Well that's not true for make, although I'm not sure of the details with ninja.

The point of suggesting to use ninja in the first place is that you can limit the number of link jobs. With 'make -j32' you can run out of memory doing a debug build of LLVM. This shouldn't happen with just 'make'.

Reply:
Are we on the same page here? Currently the directions I wrote are to just use `make` and `ninja` with no `-j` flag. I'm not sure if you're wanting to keep one or both of them as `-j32`?